lib/oneaws/cli.rb

```diff
@@ -51,5 +51,21 @@ def getkey
         EOS
       end
     end
+
+    private
+
+    # AWS の credential を以下の順番で存在チェックをする
+    # 1. ~/.aws/credentials
+    # 2. ~/.aws/config/credentials
```

Comment:
パスが違うっぽい？ :thinking: 
```suggestion
    # 2. ~/.config/aws/credentials
```

Reply:
直後に気がついて直しました。